readme.md

```diff
@@ -0,0 +1,16 @@
+Welcome to the Documentation of our Technophobe project!
+
+Aim of the project: create a database of zoo animals, function, and api endpoint to manipulate this database.
+
+Files:
+- model.py: here you find the different classes: specimen, species and genus. Genus is the parent of species, which is the parent of specimen. Use these classes when you want to create a new entry in the database for example. 
+- import_functions.py: here we have the functions that are needed to add the data to our database. 
+- data.py: in this file we have some data and we use the functions created in import_functions file to add the data to our database.  
```

Comment:
I think I might phrase it "All data on the animals is stored in this file" or something like that.